### PR TITLE
i18n refactor

### PR DESCRIPTION
**Pull request by sorja**
remove i18n from store

### CODE REVIEW

File: webapp/hooks/useI18n.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
+import i18nInstance from '@common/i18n/i18nInstance'
 import { i18n } from 'i18next'
-
+import { useEffect, useState } from 'react'
 import { useAppSelector } from '@webapp/store'
 
-export default (): i18n => useAppSelector((state) => state.app.i18n)
+export default (): Partial<i18n> => {
+  const [instance, setInstance] = useState(i18nInstance.getInstance())
```
Comment (minotogna) [inline webapp/hooks/useI18n.ts:7]: why adding to local components state the i18n instance ?
have you checked react i18 next? https://react.i18next.com

File: webapp/store/page/dataExport/reducer.ts [hunk 2]
```diff
@@ -16,7 +14,7 @@ const initialState: DataExportState = {
 }
 
 const actionHandlers = {
-  [AppActionTypes.appCountryIsoUpdate]: () => ({ ...initialState }),
+  'app/updateCountryIso': () => ({ ...initialState }),
```
Comment (minotogna) [inline webapp/store/page/dataExport/reducer.ts:17]: can we avoid hardcoded strings?

Comment (sorja) [reply webapp/store/page/dataExport/reducer.ts:17]: In dataExport refactor will be handled, app throws error if `<action>.type` is used

**Comment by sorja** [comment]
Closed in favor of react-i18next

**Pull request closed without merging**